src/assets/css/style.css

```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: #d9d9d9;
```

Comment:
I suggest that short css definitions like this one could be written on one line e.g body{xx:cc;}
Makes the files less cluttered, the bigger it gets.

Reply:
@mubstimor I think what Lillian did is the right convention in my understanding.